jobs/cach_jobs.sh

```
@@ -1,18 +1,18 @@
 #!/bin/bash
 
 # Create temporary files with lists of existing articles for all languages
-$HOME/local/bin/python3 core8/pwb.py mdpages/cashwd
+$HOME/local/bin/python3 core8/pwb.py td_core/mdpages/cashwd
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)